telemetry/packages/public-app/components/guage.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+import { useQuery } from 'react-query';
+import { Chart } from 'react-google-charts';
+import { Card } from '@tremor/react';
+
+function getRandomNumber() {
+  return Math.random() * 100;
+}
+
+export function getData() {
+  return [
+    ['Label', 'Value'],
+    ['Pressure', getRandomNumber()],
+    ['Acceleration', getRandomNumber()],
+    ['Thermometer', getRandomNumber()],
+  ];
+}
+
+export function Gauge() {
+  const [data1, setData] = useState(getData);
+  const [widthX, setWidth] = useState<number | undefined>(
```

Comment:
```suggestion
  const [width, setWidth] = useState<number | undefined>(
```

Best to stick with naming convention for `useState` and have `x` and `setX`.